src/Shell.zig

```diff
@@ -0,0 +1,303 @@
+//! Collection of utilities for scripting: an in-process sh+coreutils combo.
```

Comment:
> I need to merge this with client/shutil.zig before merging

Are you still planning to do so?

Reply:
>Are you still planning to do so?

No, upon a closer look, `shutil` does a different thing --- it concatenates strings into a single .sh or .bat script, for the purpose of testing sh/bat fragments from our readme.